src/components/Edge-Conn-PCIe.stanza

```diff
@@ -0,0 +1,943 @@
+        [pcie.data.lane[4].TX.N | B34 | Left | "Input"]
+        [pcie.data.lane[4].RX.P | A35 | Right | "Output"]
+        [pcie.data.lane[4].RX.N | A36 | Right | "Output"]
+        [pcie.data.lane[5].TX.P | B37 | Left | "Input"]
+        [pcie.data.lane[5].TX.N | B38 | Left | "Input"]
+        [pcie.data.lane[5].RX.P | A39 | Right | "Output"]
+        [pcie.data.lane[5].RX.N | A40 | Right | "Output"]
+        [pcie.data.lane[6].TX.P | B41 | Left | "Input"]
+        [pcie.data.lane[6].TX.N | B42 | Left | "Input"]
+        [pcie.data.lane[6].RX.P | A43 | Right | "Output"]
+        [pcie.data.lane[6].RX.N | A44 | Right | "Output"]
+        [pcie.data.lane[7].TX.P | B45 | Left | "Input"]
+        [pcie.data.lane[7].TX.N | B46 | Left | "Input"]
+        [pcie.data.lane[7].RX.P | A47 | Right | "Output"]
+        [pcie.data.lane[7].RX.N | A48 | Right | "Output"]
+        if width >= 16 :
+          [RSVD[1] | A50 | Right | "Passive"]
+          [pcie.data.lane[8].TX.P | B50 | Left | "Input"]
+          [pcie.data.lane[8].TX.N | B51 | Left | "Input"]
+          [pcie.data.lane[8].RX.P | A52 | Right | "Output"]
+          [pcie.data.lane[8].RX.N | A53 | Right | "Output"]
+          [pcie.data.lane[9].TX.P | B54 | Left | "Input"]
+          [pcie.data.lane[9].TX.N | B55 | Left | "Input"]
+          [pcie.data.lane[9].RX.P | A56 | Right | "Output"]
+          [pcie.data.lane[9].RX.N | A57 | Right | "Output"]
+          [pcie.data.lane[10].TX.P | B58 | Left | "Input"]
+          [pcie.data.lane[10].TX.N | B59 | Left | "Input"]
+          [pcie.data.lane[10].RX.P | A60 | Right | "Output"]
+          [pcie.data.lane[10].RX.N | A61 | Right | "Output"]
+          [pcie.data.lane[11].TX.P | B62 | Left | "Input"]
+          [pcie.data.lane[11].TX.N | B63 | Left | "Input"]
+          [pcie.data.lane[11].RX.P | A64 | Right | "Output"]
+          [pcie.data.lane[11].RX.N | A65 | Right | "Output"]
+          [pcie.data.lane[12].TX.P | B66 | Left | "Input"]
+          [pcie.data.lane[12].TX.N | B67 | Left | "Input"]
+          [pcie.data.lane[12].RX.P | A68 | Right | "Output"]
+          [pcie.data.lane[12].RX.N | A69 | Right | "Output"]
+          [pcie.data.lane[13].TX.P | B70 | Left | "Input"]
+          [pcie.data.lane[13].TX.N | B71 | Left | "Input"]
+          [pcie.data.lane[13].RX.P | A72 | Right | "Output"]
+          [pcie.data.lane[13].RX.N | A73 | Right | "Output"]
+          [pcie.data.lane[14].TX.P | B74 | Left | "Input"]
+          [pcie.data.lane[14].TX.N | B75 | Left | "Input"]
+          [pcie.data.lane[14].RX.P | A76 | Right | "Output"]
+          [pcie.data.lane[14].RX.N | A77 | Right | "Output"]
+          [pcie.data.lane[15].TX.P | B78 | Left | "Input"]
+          [pcie.data.lane[15].TX.N | B79 | Left | "Input"]
+          [pcie.data.lane[15].RX.P | A80 | Right | "Output"]
+          [pcie.data.lane[15].RX.N | A81 | Right | "Output"]
+          [RSVD[0] | B82 | Right | "Passive"]
+
+  ; pin-properties :
```

Comment:
Any reason to keep these commented out lines ?

Reply:
Nope -- oversight.